topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -0,0 +1,523 @@
+---
+layout: tutorial_hands_on
+
+title: Pseudobulk Analysis with Decoupler and EdgeR
+zenodo_link: https://zenodo.org/records/13929549
+questions:
+- How does pseudobulk analysis help in understanding cell-type-specific gene expression changes?  
+- What steps are required to prepare single-cell data (e.g., clustering, annotation, and metadata addition) for pseudobulk analysis?  
+- How can we use pseudobulk data prepared with Decoupler to perform differential expression analysis using edgeR in Galaxy?  
+objectives:
+- Understand the principles of pseudobulk analysis in single-cell data
+- Understand and generate the pseudobulk expression matrix with Decoupler
+- Perform differential expression analysis using edgeR
+time_estimation: 3H
+key_points:
+- Pseudobulk analysis approach bridges the gap between single-cell and bulk RNA-seq data  
+- Decoupler tool generates a pseudobulk count matrix, enabling downstream differential expression and functional analyses 
+- edgeR is a robust tool for differential expression in pseudobulk datasets  
+contributors:
+- dianichj
```

Comment:
We have a nicer way to add contributions now (supporting roles like authorship, reviewing, editing, funding etc):

```suggestion
contributions:
  authorship:
    - dianichj
```